catalog-rest-service/src/main/resources/json/schema/type/tagLabel.json

```diff
@@ -5,10 +5,15 @@
   "description": "This schema defines the type for labeling an entity with a Tag.",
   "type": "object",
   "javaType": "org.openmetadata.catalog.type.TagLabel",
-  "properties": {
+  "definitions": {
```

Comment:
No big changes here - moving a property into a definition so it can be reused in accessControl/rule.json schema